draft-ietf-quic-http.md

```diff
@@ -300,9 +301,9 @@ component of the URL.
 If a server presents a valid certificate and proof that it controls the
 corresponding private key, then a client will accept a secured TLS session with
 that server as being authoritative for all origins with the "https" scheme and a
-host identified in the certificate.  The host must be listed either as the CN
-field of the certificate subject or as a dNSName in the subjectAltName field of
-the certificate; see {{!RFC6125}}.  For a host that is an IP address, the client
+host identified in the certificate.  A host identified by DNS name MUST be
+validated either as a CN-ID or a DNS-ID in the certificate; see {{!RFC6125}}.
```

Comment:
IMPORTANT: I'm not sure that we need to elevate CN-ID to MUST.  In doing so, we invalidate what CABF has done: 7.1.4.2.2 clause a of the Baseline Requirements marks subject:commonName as deprecated and discouraged.  My understanding (which I can confirm) is that browsers do not use CN-ID (concretely, the CN field of the certificate subject) and so we might want to just mention the dNSName type of subjectAltName here.  We can *mention* that some clients might use subject:commonName, but that this is a deprecated practice.

I realize that this likely breaks new ground for the RFC series, and it likely needs more disucsion, but maybe we can keep up with the times with this document.

(I apologize for not having caught this earlier; I think that this is copied from elsewhere and that too needs a refresh.)

Reply:
Yes, I noticed that as well but was not about to slip it into an "editorial" PR :)

Thanks for filing the issue; I will reference it from my ballot position.
There was another point relating to certificate validation that took me a while to collect my thoughts on and write up, and I want to go over the writeup again before balloting, so I didn't get a chance to mention this topic yet.